tests/test_crypto_utils.c

```diff
@@ -37,67 +40,60 @@ const char* device_id = "470010171566423";
 const uint8_t key[32] = {82,  142, 184, 64,  74, 105, 126, 65,  154, 116, 14,  193, 208, 41,  8,  115,
                          158, 252, 228, 160, 79, 5,   167, 185, 13,  159, 135, 113, 49,  209, 58, 68};
 
-int main(void) {
-  // TODO:prefer to use c unittest framework
+void setUp(void) {}
```

Comment:
Shall we stick on camelCase such as `setUp` and `tearDown`?

Reply:
They are the functions offered by **Unity**.
Although they are camelCase, I think we should leave them unchanged.